tljh_repo2docker/__init__.py

```diff
@@ -176,13 +196,18 @@ def tljh_custom_jupyterhub_config(c):
         c.JupyterHub.spawner_class = Repo2DockerSpawner
 
         # spawner
+        c.DockerSpawner.allowed_images = "*"
```

Comment:
This was the required change to fix the failing test: https://jupyterhub-dockerspawner.readthedocs.io/en/latest/changelog.html#id5

![image](https://github.com/user-attachments/assets/9524b578-9f7f-4bf6-a3d3-a7e74054a33c)